public/components/main/report_definitions_table.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import {
+  EuiLink,
+  EuiInMemoryTable,
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiText,
+} from '@elastic/eui';
+
+export const scheduled_report_columns = [
+  {
+    field: 'reportName',
+    name: 'Name',
+    sortable: true,
+    truncateText: true,
+  },
+  {
+    field: 'type',
+    name: 'Type',
+    sortable: true,
+    truncateText: false,
+  },
+  {
+    field: 'owner',
+    name: 'Owner',
+    sortable: true,
+    truncateText: false,
+  },
+  {
+    field: 'source',
+    name: 'Source',
+    render: (username: string) => (
+      <EuiLink href={`https://github.com/${username}`} target="_blank">
```

Comment:
why use github.com user link here?

Reply:
It was the default link from the Eui docs- can replace